### PR TITLE
Migration 7-8 - Encoding key filenames with base32

### PR DESCRIPTION
**Pull request by AuHau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,8 @@ ipfs repo version | go-ipfs versions
                 4 |  0.4.3 - 0.4.5
                 5 |  0.4.6 - 0.4.10
                 6 |  0.4.11 - 0.4.15
-                7 |  0.4.16 - current
+                7 |  0.4.16 - 0.4.19
+                8 |  0.4.19 - current
```
Comment (AuHau) [inline README.md]: Depends if https://github.com/ipfs/go-ipfs/pull/6012 will make it to `0.4.19` release.

File: README.md [hunk 2]
```diff
@@ -44,7 +44,8 @@ ipfs repo version | go-ipfs versions
                 4 |  0.4.3 - 0.4.5
                 5 |  0.4.6 - 0.4.10
                 6 |  0.4.11 - 0.4.15
-                7 |  0.4.16 - current
+                7 |  0.4.16 - 0.4.19
+                8 |  0.4.20 - current
```
Comment (AuHau) [inline README.md]: Depends if ipfs/go-ipfs#6012 will make it to `0.4.20` release.

**Comment by hsanjuan** [comment]
Not sure why this adds the base32 package directly here?

**Comment by AuHau** [comment]
Well I followed https://github.com/ipfs/fs-repo-migrations#dependencies and examples of other migrations. Is there a better approach in your mind?

**Comment by hsanjuan** [comment]
> Well I followed https://github.com/ipfs/fs-repo-migrations#dependencies and examples of other migrations. Is there a better approach in your mind?

ah, ok. Yeah, should be vendored in its own folder at least. No worries, I'll take care.

**Comment by hsanjuan** [comment]
Moving to #96

**Pull request closed without merging**